### ISSUE TITLE
Make PUB_K1_xxx the default key format

### ISSUE THREAD

**Issue by cc32d9** [AntelopeIO/leap#1369]
It will be beneficial if the node software and command-line tools used the neutral notation by default, and old notation (EOSxxxx) as an option (a command-line and API parameter can specify that the client wants the legacy notation).

**Comment by bhazzard** [comment]
Agreed. We'll have to consider the implications of this as a breaking change, and the relative priority of this change.
